Tests/UnitTests/FileShareClientTests/DownloadFilesTests.cs

```diff
@@ -126,5 +127,116 @@ public async Task TestDownloadFileSetsAuthorizationHeader()
             Assert.AreEqual("bearer", fakeHttpClientFactory.HttpClient.DefaultRequestHeaders.Authorization.Scheme);
             Assert.AreEqual(DUMMY_ACCESS_TOKEN, fakeHttpClientFactory.HttpClient.DefaultRequestHeaders.Authorization.Parameter);
         }
+
+
+        [Test]
+        public async Task TestBasicDownloadFileWithCancellationToken()
+        {
+            var batchId = Guid.NewGuid().ToString();
+            string destDownldLocation = Path.Combine(Environment.CurrentDirectory, "AFilename.txt");
```

Comment:
Please change all the new tests to use a MemoryStream for the destinationStream

Reply:
changes done. Thanks